### PR TITLE
Prompt Caching support for OpenAI

### PR DESCRIPTION
**Pull request by marlenc3**

### CODE REVIEW

File: fern/docs/content/change-log/2024-10.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Changelog | October, 2024
+---
+
+## Prompt Caching Support for OpenAI
+
+OpenAI recently released some new support for [Prompt Caching](https://openai.com/index/api-prompt-caching/) with certain models.
+With this update, you'll now see the number of Prompt Cache Tokens used by a Prompt Deployment's executions if it's backed by an OpenAI model.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-10.mdx:8]:
```suggestion
Today OpenAI introduced [Prompt Caching](https://openai.com/index/api-prompt-caching/) for GPT-4o and o1 models. Subsequent invocations of the same prompt will produce outputs with lower latency and up to 50% reduced costs.

To follow this, we've begun capturing cache tokens in Vellum's monitoring layer. With this update, you'll now see the number of Prompt Cache Tokens used by a Prompt Deployment's executions if it's backed by an OpenAI model.
```

**Review by noanflaherty — APPROVED** [review]
So fast! 💨

**Comment by noanflaherty** [comment]
[oos] we should ticket and plan to update cost tracking logic to account for the 50% reduction in costs

**Comment by marlenc3** [comment]
> [oos] we should ticket and plan to update cost tracking logic to account for the 50% reduction in costs

@noanflaherty  for sure! it'll be looped in with anthropic prompt caching cost support too

**Pull request merged**